apps/wallet/tailwind.config.js

```diff
@@ -21,12 +21,15 @@ module.exports = {
             minHeight: {
                 8: '2rem',
             },
+            margin: {
+                7.5: '1.875rem',
```

Comment:
Just add it to spacing and it should be added everywhere.

Reply:
done